src/modelmanager.cpp

```diff
@@ -586,7 +586,7 @@ Status ModelManager::loadMetricsConfig(rapidjson::Document& configJson) {
     } else {
         const auto& metrics = itr2->value.GetObject();
         SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Parsing monitoring metrics config settings.");
-        firstErrorStatus = this->metricConfig.parseMetricsConfig(metrics);
+        firstErrorStatus = this->metricConfig.parseMetricsConfig(metrics, ovms::Config::instance().restPort());
```

Comment:
Why do we need to pass port to `parseMetricsConfig`? We pass down the responsibility to parser method. We chould keep this responsibility here in ModelManager by just returning error in line 587 if rest port is 0.